pom.xml

```diff
@@ -0,0 +1,85 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.narayana</groupId>
+    <artifactId>narayana-openshift-tools-parent</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0.0.Final-SNAPSHOT</version>
+    <name>Narayana OpenShift tools - parent</name>
+
+    <distributionManagement>
+      <repository>
+        <id>jboss-releases-repository</id>
+        <name>JBoss Release Repository</name>
+        <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+      </repository>
+      <snapshotRepository>
+        <id>jboss-snapshots-repository</id>
+        <name>JBoss Snapshot Repository</name>
```

Comment:
did you try a snapshot deploy? I think you should add the nexus maven plugin to make it easier